src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java

```diff
@@ -128,6 +128,31 @@ public void testInputValidation() {
         feedbackEditPage.clickAddMsqOtherOptionCheckboxForNewQuestion();
         assertTrue(feedbackEditPage.isElementSelected("msqOtherOptionFlag--1"));
         feedbackEditPage.clickAddQuestionButton();
+
+        ______TS("Check that Max/Min selectable choices cannot be blank");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("MSQ");
+        feedbackEditPage.fillQuestionTextBoxForNewQuestion("Test question text");
+        feedbackEditPage.fillQuestionDescriptionForNewQuestion("more details");
+        feedbackEditPage.fillMsqOptionForNewQuestion(0, "Choice 1");
+        feedbackEditPage.fillMsqOptionForNewQuestion(1, "Choice 2");
+
+        feedbackEditPage.toggleMsqMaxSelectableChoices(NEW_QUESTION_INDEX);
+        feedbackEditPage.fillMsqMaxSelectableChoices(NEW_QUESTION_INDEX, "");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.isInputElementValid(feedbackEditPage.getMsqMaxSelectableChoicesBoxElement(NEW_QUESTION_INDEX));
```

Comment:
`isInputElementValid` returns `True` or `False`, I think you want to do `assertTrue` or `assertFalse` here?

Reply:
Actually you can choose not to `clickAddQuestionButton()`, before checking validity of that input element.